Scripts/GetCausalCAVIAR.py

```diff
@@ -0,0 +1,170 @@
+import argparse
+import math
+import numpy as np
+import os
+import pandas as pd
+import random
+import shutil
+import sys
+import gzip
+
+"""
+Quantify the probability of a variant to be causal while allowing with arbitrary number of causal variants
```

Comment:
Not sure we will allow an arbitrary number of causal variants right? We talked about starting with -c1 for now